mobile/native/src/orderbook.rs

```diff
@@ -65,6 +66,13 @@ pub fn subscribe(secret_key: SecretKey) -> Result<()> {
                                     tracing::error!("Trade request sent to coordinator failed. Error: {e:#}");
                                 }
                             },
+                            OrderbookMsg::AllOrders(orders) => {
+                                // TODO: How to trigger this more often? this comes only initially
```

Comment:
I think the idea is that any subsequent order data will be updated through delta events.

- `OrderbookMsg::NewOrder(Order)`
- `OrderbookMsg::DeleteOrder(Uuid)`
- `OrderbookMsg::Update(Order)`

The client will then have to update her local list correspondingly.

Reply:
ouch, that's a tough one, because having that being piped through might be really heavy.

might be best to just ask for the best price in the context of all these events after all...

Reply:
As Richie said: this event is only fired on first opening the websocket connection, afterwards you only get the diff with `NewOrder`, `DeleteOrder` and `UpdateOrder`. 

Wouldn't it be easy to just maintain a list of orders in this file and whenever there is an update you check if it is worth calling `position::handler::price_update...` or not?

Reply:
great idea. I will maintain the the list of orders here - assembling the full picture out of a delta would be too laborious in flutter.